extension/app/src/components/input_bar/InputBar.tsx

```diff
@@ -19,23 +18,6 @@ import { useContext, useEffect, useMemo, useRef, useState } from "react";
 
 // AGENT MENTION
 
-function AgentMention({
```

Comment:
nit: remove comment ⬆️ 
we also have the same unused code in front/InputBar, can you remove it there too ?